addons/io/org.openhab.io.homekit/README.md

```diff
@@ -27,8 +27,80 @@ org.openhab.homekit:thermostatHeatMode=HeatOn
 org.openhab.homekit:thermostatAutoMode=Auto
 org.openhab.homekit:thermostatOffMode=Off
 org.openhab.homekit:networkInterface=192.168.0.6
+```
+
+The following additional settings can be added or edited in PaperUI after switching to expert mode:
 
 ```
+org.openhab.homekit:name=openHAB
+org.openhab.homekit:minimumTemperature=-100
+org.openhab.homekit:maximumTemperature=100
+```
+
+### Overview of all settings
+
+<table>
+ <tr>
+  <th>Setting</th>
+  <th>Description</th>
+  <th>Default value</th>
+ </tr>
+ <tr>
+  <td>networkInterface</td>
+  <td>IP address or domain name under which the HomeKit bridge can be reached. If no value is configured, the addon tries to determine the IP address from the local hostname. </td>
+  <td>(none)</td>  
+ </tr>
+ <tr>
+  <td>port</td>
+  <td>Port under which the HomeKit bridge can be reached. </td>
+  <td>9123</td>  
+ </tr>
+ <tr>
+  <td>pin</td>
+  <td>Pin code used for pairing with iOS devices. Apparently, pin codes are provided by Apple and represent specific device types, so they cannot be chosen freely. The pin code <em>031-45-154</em> is used in sample applications and known to work.</td>
+  <td>031-45-154</td>  
+ </tr>
+ <tr>
+  <td>useFahrenheitTemperature</td>
+  <td>Set to <em>true</em> to use Fahrenheit degrees, or <em>false</em> to use Celsius degrees.</td>
+  <td>false</td>  
+ </tr>
+ <tr>
+  <td>thermostatCoolMode</td>
+  <td>Word used for activating the cooling mode of the device (if applicable).</td>
+  <td>CoolOn</td>  
+ </tr>
+ <tr>
+  <td>thermostatHeatMode</td>
+  <td>Word used for activating the heating mode of the device (if applicable).</td>
+  <td>HeatOn</td>  
+ </tr>
+ <tr>
+  <td>thermostatAutoMode</td>
+  <td>Word used for activating the automatic mode of the device (if applicable).</td>
+  <td>Auto</td>  
+ </tr>
+ <tr>
+  <td>thermostatOffMode</td>
+  <td>Word used for turning thermostat mode of the device to off (if applicable).</td>
+  <td>Off</td>  
+ </tr>
+ <tr>
+  <td>minimumTemperature</td>
+  <td>Lower bound of possible temperatures, used in the user interface of the iOS device to display the allowed temperature range. Note that this setting applies to all devices in HomeKit. </td>
+  <td>-100</td>  
+ </tr>
+ <tr>
+  <td>maximumTemperature</td>
+  <td>Upper bound of possible temperatures, used in the user interface of the iOS device to display the allowed temperature range. Note that this setting applies to all devices in HomeKit. </td>
+  <td>100</td>  
+ </tr>
+ <tr>
+  <td>name</td>
+  <td>Name under which this HomeKit bridge is announced in the network and which is displayed on the iOS device when searching for available bridges. </td>
```

Comment:
Consider, "Name under which this HomeKit bridge is announced on the network.  This is also the name which is displayed..."